content/javascript/concepts/dates/terms/getUTCFullYear/getUTCFullYear.md

```diff
@@ -0,0 +1,33 @@
+---
+title: ".getUTCFullYear()"
+description: "Returns the year of the specified date according to universal time. It extracts the four-digit year (e.g., 2023) from a given Date object, based on the Coordinated Universal Time (UTC) time zone."
```

Comment:
```suggestion
Description: 'Returns the year of the specified date according to universal time.'
```

Reply:
Please update the remaining lines within the metadata section to use single quotes